### PR TITLE
Correct function index during translation

### PR DESCRIPTION
**Pull request by lum1n0us**
N from "aot_func#N" adds import function count is the correct wasm function index

### CODE REVIEW

**Comment by lum1n0us** [comment]
@johnlanni please take a look and give it a try

**Review by johnlanni — APPROVED** [review]
Testing is fine

File: doc/perf_tune.md [hunk 1]
```diff
@@ -98,17 +98,23 @@ You should only use this method for well tested wasm applications and make sure
...
 
-This will create a _perf.data_ and a _jit-xxx.dump_ under _~/.debug.jit/_ folder. This extra file is WAMR generated at runtime, and it contains the mapping between the JIT code and the original Wasm function names.
+This will create a _perf.data_ and
+- a _jit-xxx.dump_ under _~/.debug/jit/_ folder if running llvm-jit mode
+- or _/tmp/perf-<pid>.map_ if running AOT mode
 
-The next thing need to do is to merge _jit-xxx.dump_ file into the _perf.data_.
+
+This file is WAMR generated. It contains information includes jitted(precompiled) code addresses in memory, names of jitted
+(precompiled) functions which like *aot_func#N* and so on.
```
Comment (wenyongh) [inline doc/perf_tune.md:115]: `includes` -> `which includes` or `including`?

`which like` -> `which are like`

File: doc/perf_tune.md [hunk 2]
```diff
@@ -98,17 +98,23 @@ You should only use this method for well tested wasm applications and make sure
...
+
+If running llvm-jit mode, the next thing is to merge _jit-xxx.dump_ file into the _perf.data_.
```
Comment (wenyongh) [inline doc/perf_tune.md:117]: how about `running with llvm-jit mode`

File: doc/perf_tune.md [hunk 3]
```diff
@@ -141,28 +147,28 @@ $ perf report --input=perf.data
 [Flamegraph](https://www.brendangregg.com/flamegraphs.html) is a powerful tool to visualize stack traces of profiled software so that the most frequent code-paths can be identified quickly and accurately. In order to use it, you need to [capture graphs](https://github.com/brendangregg/FlameGraph#1-capture-stacks) when running `perf record`
 
 ```
-$ perf record -k mono --call-graph=fp --output=perf.data.raw -- iwasm --perf-profile foo.wasm
+$ perf record -k mono --call-graph=fp --output=perf.data.raw -- iwasm --enable-linux-perf foo.wasm
 ```
 
-merge the _jit-xxx.dump_ file into the _perf.data.raw_.
+If running llvm-jit mode, merge the _jit-xxx.dump_ file into the _perf.data.raw_.
```
Comment (wenyongh) [inline doc/perf_tune.md:153]: how about `running with llvm-jit mode`

**Review by wenyongh — APPROVED** [review]

**Pull request merged**